tests/test_uids.py

```diff
@@ -42,7 +44,7 @@ def make_mock_neuron(unique_coldkeys=False, unique_ips=False, vpermit_tao_limit=
         ])
 def test_get_random_uids(unique_coldkeys, unique_ips, k, expected_result):
```

Comment:
Function typing is very helpful here. You have also removed the idea of using `k`, so please replace `k` with `sample_size` to avoid confusion (I was confused for a bit here)